ert3/stats/_stats.py

```diff
@@ -163,6 +163,50 @@ def type(self) -> str:
         return "uniform"
 
 
+class LogUniform(Distribution):
+    def __init__(
+        self,
+        lower_bound: float,
+        upper_bound: float,
+        *,
+        size: Optional[int] = None,
+        index: Optional[ert.data.RecordIndex] = None,
+    ) -> None:
+        self._lower_bound = lower_bound
+        self._upper_bound = upper_bound
+
+        def rvs(size: int) -> np.ndarray:  # type: ignore
+            return np.array(
+                scipy.stats.loguniform.rvs(
+                    a=self._lower_bound, b=self._upper_bound, size=self._size
+                )
+            )
+
+        def ppf(x: np.ndarray) -> np.ndarray:  # type: ignore
+            return np.array(
```

Comment:
Is it an option to use `npt.NDArray` from `numpy.typing as npt` instead of `# type: ignore`?
https://numpy.org/doc/stable/reference/typing.html#module-numpy.typing

Reply:
Yes, but Python 3.6 can only use an older version of numpy that does not support that. That would still be ok, if we make strings out of those types, which is not very elegant. It seems we have not done that anywhere in ERT yet, maybe something to revisit when we have 3.8 as the minimum.